pymeasure/instruments/lecroy/lecroyT3DSO1204.py

```diff
@@ -510,10 +509,10 @@ def __init__(self, adapter, **kwargs):
         self._seconds_since_last_write = 0  # Timestamp of the last command
         self._header_size = 16  # bytes
         self._footer_size = 2  # bytes
-        self.ch1 = Channel(self, 1)
-        self.ch2 = Channel(self, 2)
-        self.ch3 = Channel(self, 3)
-        self.ch4 = Channel(self, 4)
+        self.ch_1 = ScopeChannel(self, 1)
+        self.ch_2 = ScopeChannel(self, 2)
+        self.ch_3 = ScopeChannel(self, 3)
+        self.ch_4 = ScopeChannel(self, 4)
         self.waveform_source = "C1"
         self.default_setup()
 
```

Comment:
In order to add Channels manually (should only be used for dynamic channel creation), you could call:
`self.add_child(ScopeChannel, 1`

We made it even easier. Just add to the class (similar to the properties):
```suggestion
        self.waveform_source = "C1"
        self.default_setup()

    channels = ChannelCreator(ScopeChannel, (1, 2, 3, 4))

```
This code will make the same thing, as your four lines (plus additional magic, like creating a dictionary `self.channels` containing the channels...).

Reply:
I suspect that this suggestion is referring to a previous version of the code. Could you please confirm that the current version is correct/appropriate?

Reply:
Yes, the current version is correct.